controller/space.go

```diff
@@ -164,6 +171,37 @@ func (c *SpaceController) Delete(ctx *app.DeleteSpaceContext) error {
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
 	}
+
+	// extract config to be generic to be utilized elsewhere
+	config, ok := c.config.(*configuration.Registry)
+	if !ok {
+		return fmt.Errorf("no config found")
```

Comment:
Should this be `errs.Errorf` to include the stacktrace?

Reply:
done!